docs/using-fake-timers.md

```diff
@@ -0,0 +1,34 @@
+---
+id: using-fake-timers
+title: Using Fake Timers
+sidebar_label: Using Fake Timers
+---
+
+Using real timers in your tests is less common since they depend on real time
+lapse. To solve this, most testing frameworks offer the option to use fake
+timers in your tests so you won't need to rely on real times.
```

Comment:
I think it's better to explain why real timers are bad for tests:
- flaky tests
- poor predictability
- can't make assumptions about specific timestamps.

Reply:
I wasn't sure we want to explain the rational to using fake timers since this page is for people already using it, 
that's why my explanation wasn't thorough.. I'll expand it :)

Reply:
Hmm that works as well. I wouldn't make any assumptions about how common one or the other is though.

Reply:
I agree. If I were to guess, real timers are far more common (I personally use them only very rarely) mostly because real timers are the default.

Reply:
Updated and expanded the explanation on why to use fake timers :)
Thanks!